models/staging/synthea/_synthea__models.yml

```diff
@@ -748,12 +746,6 @@ models:
       - name: uncovered_encounters
         data_type: integer
         description: "The number of Encounters not paid for by this Payer, and paid out of pocket by patients."
-      - name: covered_medications
-        data_type: integer
-        description: "The number of Medications paid for by this Payer."
-      - name: uncovered_medications
-        data_type: integer
-        description: "The number of Medications not paid for by this Payer, and paid out of pocket by patients."
```

Comment:
actually it looks like these should be in the stg table, i just forgot to include them in the sql query 😮‍💨

Reply:
Potentially! I don't think they're used downstream (yet) though?

Reply:
they're not, but for consistency i think we should just add them to the models (i also _believe_ this is the general recommendation from dbt)

Reply:
Ahh sorry I'm being thick - I see what you mean forgive me!